drivers/hmis/app/views/hmis_admin/project_groups/_form.haml

```diff
@@ -0,0 +1,23 @@
+= f.error_notification
+
+-# Project Group must have data source specified. Only allow selecting inclusion criteria for the given data source
+- ds = @project_group.data_source
+- organization_options = GrdaWarehouse::Hud::Organization.where(data_source_id: ds.id).options_for_select(user: current_user)
+
+%div.mb-4
+  %h3 Project Group Name
+  = f.input :name
+
+%div.mb-4
+  %h3 Inclusion Criteria
+  = f.simple_fields_for :inclusion_criteria do |ic|
+    = ic.input :project_ids, collection: [], as: :select_two, input_html: { multiple: true, data: { 'collection-path' => api_projects_path(selected_project_ids: @project_group.parsed_inclusion_criteria.project_ids, data_source_ids: [ds.id]) }, placeholder: 'Choose Projects'}, label: 'Projects', required: false
+    = ic.input :organization_ids, collection: organization_options, selected: @project_group.parsed_inclusion_criteria.organization_ids, as: :grouped_select_two, group_method: :last, input_html: {multiple: true, placeholder: 'Choose Organizations' }, label: 'Organizations', required: false
+    = ic.input :project_type_numbers, collection: Hmis::ProjectGroupCriteria.available_project_type_numbers, selected: @project_group.parsed_inclusion_criteria.project_type_numbers, input_html: { multiple: true, placeholder: 'Choose Project Types' }, label: 'Project Types', as: :select_two, required: false
+    = ic.input :data_source_ids, collection: GrdaWarehouse::DataSource.where(id: ds.id).options_for_select(user: current_user), selected: @project_group.parsed_inclusion_criteria.data_source_ids, as: :select_two, input_html: {multiple: true, placeholder: 'Choose Data Sources' }, label: 'Data Sources', required: false, hint: 'If selected, all projects in the Data Source will be included in the group.'
```

Comment:
Pretty sure this should only let you choose one data source, and it should be limited to the data source of this project group... which I now see is what you have, though maybe just for consistency it is a multi-select with a plural ids name.

I'm ok with that choice.

Reply:
Yeah, good point. Yes, this is a bit messy since I added the "source" data source id concept after.

I was toying with the idea that the Project Group Criteria class might be usable/useful for other future purposes that _are_ cross-data-source, so part of me wants to leave that door open in the JSON criteria shape.

Reply:
I'm working on a change to just make this an all projects flag